geonode/api/resourcebase_api.py

```diff
@@ -454,6 +454,12 @@ def get_list(self, request, **kwargs):
             collection_name=self._meta.collection_name)
         to_be_serialized = paginator.page()
 
+        bundles = [
```

Comment:
@davisc @theduckylittle not the answer you are looking for, but for reference see https://github.com/GeoNode/geonode/pull/1537